frontend_tests/puppeteer_tests/navigation.ts

```diff
@@ -97,8 +107,8 @@ async function navigation_tests(page: Page): Promise<void> {
     await navigate_to_subscriptions(page);
     await navigate_using_left_sidebar(page, "all_messages", "message_feed_container");
     await navigate_to_settings(page);
-    await navigate_using_left_sidebar(page, "narrow/is/private", "message_feed_container");
     await navigate_to_subscriptions(page);
+    await navigate_to_private_messages(page);
```

Comment:
Can you move this above subscriptions so that it's still in the same order?

Reply:
Oops, sorry I mispositioned it there, Updated the PR.